Sources/SwiftGRPCNIO/GRPCServer.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+import NIO
+import NIOHTTP1
+import NIOHTTP2
+
+// Wrapper object to manage the lifecycle of a gRPC server.
+public final class GRPCServer {
+  public static func start(
```

Comment:
Can we add headerdocs for these and other public interfaces? I think that'd greatly help with documentation/clarity

Reply:
Grat idea! Done.